utils/src/main/scala/za/co/absa/enceladus/utils/schema/SparkUtils.scala

```diff
@@ -104,4 +86,7 @@ object SparkUtils {
     dfWithAggregatedErrColumn.drop(tmpColumn)
   }
 
+  def ifExistsErrorFunction(colExpr: Column): (DataFrame, String) => DataFrame =
```

Comment:
I think the function name is more confusing than helping, took my a while to understand. 
I would prefer using the `overwriteWithErrorColumn(_, _, colExpr)` as partially defined.
And base on the observation of usage perhaps create a function replacing `withColumnIfDoesNotExist(overwriteWithErrorColumn)...` with something like `withColumnOverwrightIfExists(colName, colExpr)`.

Reply:
Done